.github/workflows/build-text.yml

```
@@ -0,0 +1,52 @@
+name: Build Website - Text
+on:
+  push:
+    branches:
+      - master
+      - dev
+
+concurrency:
+  group: build-text-doc-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    uses: ./.github/workflows/build-branch.yml
+    with:
+      branch: ${{ github.ref }}
+      num_workers: 10
```

Since we are not executing demos, I don't think there is a need to have such a high number of workers ... maybe we can do 2 or 3? I think we can even do 1, since it take less than 5 min to build on single thread when not executing demos.

---

Decrease number of workers to `1`.
```suggestion
      num_workers: 1
```